lucene/codecs/src/java/org/apache/lucene/codecs/blocktreeords/OrdsSegmentTermsEnum.java

```diff
@@ -1082,12 +1082,14 @@ private InputOutput getByOutput(long targetOrd) throws IOException {
       if (FST.targetHasArcs(arc)) {
         // System.out.println("  targetHasArcs");
         result.grow(1+upto);
-        
+        if (arc.target < 0 || arc.target > Integer.MAX_VALUE) {
```

Comment:
What's the point of this block (and isn't it effectively dead code)?

Reply:
Thanks for the comments, and sorry for the messes you found! It looks as if I fumbled uploading the patch and pushed a slightly outdated version that was missing some final cleanups, removing debug code and the like. I think failed to keep changes in sync between this and the working version from our production system that still builds against 7.5. Anyway I'll address that soon.

Reply:
@dweiss I think I addressed your comments and this is ready to be committed. If you agree, would you be able to help with that (I'm not a committer, so I need help!)?